src/web/styles/themes.scss

```diff
@@ -35,6 +35,7 @@
   --theme-disabled-button-text: #878787;
   --theme-disabled-button: #d9d9d9;
   --theme-label: rgba(120, 204, 0, 0.4);
+  --theme-error-label: rgba(253, 4, 4, 0.25);
```

Comment:
This should probably be theme-error-label-background to be consistent with the other background colours.